<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionsTorston</class>
 <widget class="QWidget" name="QuestionsTorston">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#QuestionsTorston{
	background-image: url(&quot;:/pictures/back.png&quot;);
}</string>
  </property>
  <widget class="QLabel" name="esc">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>*Можно использовать клавиши клавиатуры (1, 2, 3, 4, 5), для выхода нажмитие &lt;Esc&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>18</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Справка - F1</string>
   </property>
  </widget>
  <widget class="QWidget" name="Title" native="true">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>45</y>
     <width>503</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	border-radius: 10px;
	background-color:rgba(217, 217, 217, 1);
}</string>
   </property>
   <widget class="QTextEdit" name="TitleTorston">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat Medium'; font-size:18pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;ТЕСТ ТОРСТОНА&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="SectionTorston" native="true">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>126</y>
     <width>893</width>
     <height>334</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#SectionTorston {
	border-radius: 10px;
	background-color:rgba(217, 217, 217, 1);
}</string>
   </property>
   <widget class="QLabel" name="label_question">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_picture" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>588</width>
      <height>94</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_picture {
	background-image: url(C:/workspace/PsychologyTestApp/tests/Torston/images/Fig0.bmp);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="choiceTorston">
    <item>
     <widget class="QPushButton" name="button5">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(217, 217, 217);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}
QPushButton:checked {
        background-color: rgba(187, 147, 148);
		color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>01</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button4">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(217, 217, 217);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}
QPushButton:checked {
        background-color: rgba(187, 147, 148);
		color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>02</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button3">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(217, 217, 217);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}
QPushButton:checked {
        background-color: rgba(187, 147, 148);
		color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>03</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button2">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(217, 217, 217);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}
QPushButton:checked {
        background-color: rgba(187, 147, 148);
		color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>04</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(217, 217, 217);
}
QPushButton:hover {
        background-color: rgba(216, 249, 255);
}
QPushButton:checked {
        background-color: rgba(187, 147, 148);
		color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>05</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>60</y>
     <width>118</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nextQuestion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>761</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: rgb(79, 121, 66);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
        background-color: rgba(187, 147, 148);

}</string>
   </property>
   <property name="text">
    <string>Отправить ответ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
